docs/superpowers/plans/2026-07-10-nfd-0.19.0-bump.md

```
@@ -0,0 +1,334 @@
+- Modify: `recipes/registry.yaml:53`
+- Modify: `recipes/overlays/base.yaml:31`
+- Modify: `recipes/components/nfd/values.yaml:15-19`
+
+**Interfaces:**
+- Consumes: clean worktree at `527c015e0` (plus the two committed docs).
+- Produces: all three recipe files reference v0.19.0; `enableNodeFeatureApi` no longer exists as a live key anywhere; remaining references (nfd-ocp comment, chainsaw asserts) are removed by the final-review fix commit. Task 2's BOM regen reads the bumped registry pin.
+
+- [ ] **Step 1: Pre-check (RED analog for config): confirm 0.19.0 absent and current state matches plan**
+
+Run from the worktree root:
+```bash
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should be surrounded by blank lines

(MD031, blanks-around-fences)